b/bootstrap-select/bootstrap-select_ubi_8.5.sh

```diff
@@ -0,0 +1,57 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bootstrap-select
+# Version       : v1.12.4
+# Source repo   : https://github.com/snapappointments/bootstrap-select
+# Tested on	    : "Red Hat Enterprise Linux 8.5" (Docker)
```

Comment:
Can we change this to UBI?
```suggestion
# Tested on		: UBI 8.5
```

Reply:
@Siddhesh-Ghadi  Updated, Please review.